src/Uno.Sdk/targets/winappsdk-workaround.targets

```diff
@@ -0,0 +1,21 @@
+<Project>
+	<!-- Workaround to avoid including Project XBFs in the PRI file -->
+	<Target Name="AdjustGetPackagingOutput1" AfterTargets="GetMrtPackagingOutputs">
```

Comment:
Let's add an explicit `DisableWinUI8857_Workaround` so we can override the behavior as needed.